CourseDevelopment/Week5/Chapter5.MD

```diff
@@ -0,0 +1,99 @@
+After this week, one understands the role, responsibilities and interaction of scientists with scientific communities and the general public. 
+Students have to give a presentation about their other assignments using skills learned in the course. 
+Further, the studends are able to formulate their opinion in a discussion on the role a scientist must take in a public debate.
+
+## Keywords student familiarity check week 5
+Some of the keywords in syllabs.MD are supposed to be new, while some are most probably familiar for students.
+
+|**New Concepts**|**Familiar Concepts**|
+|----------------|---------------|
+|How storytelling works|Metaphors|
+|Science communication and journalism|Communicating uncertainty|
+|Activism||
+|trust in science||
+|Bounds of expertise||
+|Academic social contract||
+|Narratives and framing||
+|Difficult to reach audience||
+|Process and outcome||
+
+
+## Lecture: Week 5
+
+**Preparation:** Read article about how physics is represented in Dutch newspapers. https://jcom.sissa.it/archive/20/07/JCOM_2007_2021_A02
+
+**Part 1** A short “classic” lecture of 25 min on:\
+Academic social contract\
+Bounds of expertise\
+Naratives and framing\
+trust in science\
+Difficulties to reach audience\
+process and outcome\
+trust in science\
+
+**Part 2** Guest lecture & Discussion (60 min)\
+On representation of physics in the Netherlands\
+Lecture by e.g. Julia Cramer (Leiden), followed up by a plenary discussion.
+
+
+## Tutorial Week 5: What is the role of a scientist in society?\
+(2x 60 min)
+\
+**Topics in tutorial:**\
+Metaphors | Communicating uncertainty | possible roles of a scientist
+
+### Tutorial format
++ **Assignment objective**
+Students practice presenting and express their points of view, by reflecting on their role as a future scientist. They will use metaphors to communicate topics of their bachelor research. 
+
++ **Possible Assignments**
+Students can choose between:
+++ *A plenary discussion*\
+Students are devided in groups of 8. In the first our of the tutorial session, each group will prepare 5 statements for a plenary debate. 
+The statements should anwer questions such as: ...\
+**TODO:** what type of questions need to be answerd?\
+For every statement, their should be arguments formulated aganst it and in favor of the statement.\
+
+In the second hour of the tutorial all statements go in a (virtual) jar. Five of them will be selected at random.
+After that, the group bringing in this statement will introduce it in a few words and the debate wil start (10 min). 
+To keep the discussion running, the prepered arguments can be trown in.
+
+
+++ *A Pitch*\
+Students present a pitch of 3 min, to explain why, their bachelor thesis is relevant for society. 
+The first hour can be used for preparations. In the second part, all pitchers come to gether to do their pitch. 
+\
+\
+**TODO:** How should these assignments be graded?
```

Comment:
Weren't we going for pass/fail throughout? So that any student who shows reasonable participation in the tutorial will get a pass?

Reply:
A well, that will solve the grading problem! I agree with the pass/fail.

For the students to get more insight in their performence, teachers can provide them with some feedback. What do you think?